src/components/Home/Keshav/Week5/Counter/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { CounterWrapper } from './styles';
+import { useSelector, useDispatch } from 'react-redux';
+import { incrementCounter, decrementCounter } from '../redux/actions/CounterActions';
+import { RootState } from '../redux/actions/types';
+
+const Counter: React.FC = () => {
+  const count = useSelector((state: RootState) => state.count);
+  const dispatch = useDispatch();
+
+  const increment = () => {
+    dispatch(incrementCounter());
+  };
+
+  const decrement = () => {
+    dispatch(decrementCounter());
+  };
+
+  return (
+    <CounterWrapper>
+      <h1>Counter using Redux</h1>
+      <span className='counterHeading'>Counter : {count}</span>
+      <br />
+      <br />
```

Comment:
use css to add spacing instead of br tags

Reply:
I have tried giving margin-bottom directly to span tag, but content was not coming below to it.